stanford/bin/audit-iam-users

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env python
+# pylint: disable=invalid-name
+# pylint: disable=missing-docstring
+from __future__ import print_function
```

Comment:
This is to use Python3's print function, right?

Reply:
https://docs.python.org/2/library/__future__.html
https://www.python.org/dev/peps/pep-3105/